### PR TITLE
Feature/dynamodb

### PR DESCRIPTION
**Pull request by christodo**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -1,136 +1,71 @@
 <?xml version='1.0' encoding='utf-8'?>
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
```
Comment (mithie) [inline pom.xml]: please add dev/prod profiles for local and AWS deployment

File: src/main/java/com/bmw/remotecollab/admin/rest/SessionController.java [hunk 2]
```diff
@@ -59,6 +60,7 @@ public SessionController(OpenViduService openViduService, RoomService roomServic
     @PostMapping("/room/join")
```
Comment (mithie) [inline src/main/java/com/bmw/remotecollab/admin/rest/SessionController.java]: please think about REST compliant resource name

Comment (christodo) [reply src/main/java/com/bmw/remotecollab/admin/rest/SessionController.java]: will be done after our mvp

File: pom.xml (hunk 1 above)
Comment (christodo) [reply pom.xml]: We will do it in a separate feature branch

**Pull request merged**